public/app/plugins/datasource/loki/datasource.ts

```diff
@@ -283,8 +283,12 @@ export class LokiDatasource
     return query.expr;
   }
 
+  getTimeRange() {
+    return this.timeSrv.timeRange();
+  }
+
   getTimeRangeParams() {
-    const timeRange = this.timeSrv.timeRange();
+    const timeRange = this.getTimeRange();
```

Comment:
do you want to use `this.getTimeRange()` at https://github.com/grafana/grafana/blob/matyax/monaco-loki-from-hackathon/public/app/plugins/datasource/loki/datasource.ts#L259 too?